mgr/v0.2.0-5.34.38-64b.csh

```diff
@@ -0,0 +1,2 @@
+module use /cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86_64/
```

Comment:
One more item to note / consider... this scheme assumes that we are always establishing an environment under linux-rhel7-x86_64.... how do we support different versions of linux?

Reply:
resolved with b9a24df